ModelBundle/DataFixtures/MongoDB/LoadContentData.php

```diff
@@ -155,29 +150,41 @@ public function generateCar206($language)
      */
     public function generateCustomerConvenant($language)
     {
-        $content = new Content();
-
-        $attribute1 = $this->generateContentAttribute('firstname', 'Jean-Claude');
-        $attribute2 = $this->generateContentAttribute('lastname', 'Convenant');
-        $attribute3 = $this->generateContentAttribute('identifier', 28);
-
-        $content->setContentId("jean_paul");
-        $content->setContentType("customer");
-        $content->setContentTypeVersion(1);
-        $content->setDeleted(false);
+        $content = $this->addBaseContent("jean_paul", 'customer', 1);
         $content->setName("Jean-Paul");
         $content->setLanguage($language);
-        $content->setStatus($this->getReference('status-published'));
         $content->setVersion(2);
         $content->addKeyword(EmbedKeyword::createFromKeyword($this->getReference('keyword-lorem')));
         $content->addKeyword(EmbedKeyword::createFromKeyword($this->getReference('keyword-sit')));
         $content->setLinkedToSite(false);
         $content->setSiteId('2');
 
+        $attribute1 = $this->generateContentAttribute('firstname', 'Jean-Claude');
+        $attribute2 = $this->generateContentAttribute('lastname', 'Convenant');
+        $attribute3 = $this->generateContentAttribute('identifier', 28);
+
         $content->addAttribute($attribute1);
         $content->addAttribute($attribute2);
         $content->addAttribute($attribute3);
 
         return $content;
     }
+
+    /**
+     * @param $id
```

Comment:
Could you :
 - type the parameters
 - align the parameters
 - Set an empty line between @param and @return